src/features/Classes/ClassDetailPage/InstructorCard.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { formatDateRange } from 'react-paragon-topaz';
+import { Spinner, ProgressBar } from '@edx/paragon';
+
+import { RequestStatus } from 'features/constants';
+
+const InstructorCard = () => {
+  const classes = useSelector((state) => state.common.allClasses);
+  const [classInfo] = classes.data;
+
+  const isLoadingClasses = classes.status === RequestStatus.LOADING;
+
+  const totalEnrolled = (classInfo?.numberOfStudents || 0)
+    + (classInfo?.numberOfPendingStudents || 0);
+
+  const percentageEnrolled = totalEnrolled && classInfo?.maxStudents
+    ? ((totalEnrolled * 100) / classInfo.maxStudents)
+    : 0;
+
+  return (
+    <article className="instructor-wrapper mb-4 d-flex flex-column flex-sm-row justify-content-between align-items-start page-content-container">
+      <div className="d-flex flex-column w-100 justify-content-between w-100">
+        {isLoadingClasses && (
+          <div className="w-100 h-100 d-flex justify-content-center align-items-center">
+            <Spinner
+              animation="border"
+              className="mie-3"
+              screenReaderText="loading"
+            />
+          </div>
+        )}
+        {!isLoadingClasses && (
+          <>
+            <h5 className="text-color text-uppercase font-weight-bold text-truncate w-75" title={classInfo?.masterCourseName}>
```

Comment:
This should have more space at the top and bottom, the distance between header -> title <- card, should be the same.

![image](https://github.com/user-attachments/assets/aebf4f81-c38e-40ba-9488-53bd68f920f1)